package.json

```diff
@@ -14,6 +14,7 @@
     "url": "https://github.com/oqton/redux-black-box.git"
   },
   "main": "lib.js",
+  "typings": "lib.d.ts",
```

Comment:
i _think_ this should be `types` : https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
ah both are supported